tests/template/create_template_test.go

```diff
@@ -0,0 +1,192 @@
+package template_test
+
+import (
+	"os"
+	"strings"
+
+	"github.com/nimblehq/gin-templates/tests"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = BeforeEach(func() {
+	tests.ChangeDirectory(TemplateGeneratedPath + "/test-gin-templates")
+})
+
+var _ = Describe("Create template", func() {
+	Context("given current directory", func() {
```

Comment:
How about:

```suggestion
	Context("given the project directory", func() {
```

Reply:
Fixed in bf2e40575707b1ee2d9871cd1c51202e92e25fd9